proto/uber/cadence/api/v1/service_domain.proto

```diff
@@ -114,6 +114,7 @@ message DescribeDomainRequest {
 
 message DescribeDomainResponse {
   Domain domain = 1;
+  FailoverInfo failover_info = 2;
```

Comment:
Any reason why this is added to `DescribeDomainResponse` instead directly to `Domain`?
If we were to add it to `Domain`, other API call `ListDomains` would also get this additional info. Or is it intentionally left out there?